tests/album-update.test.js

```diff
@@ -0,0 +1,76 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Update Album', () => {
+    let artist;
+    let album;
+    let singleArtistId;
+    beforeEach(async () => {
+        let albumData;
+        let artistData;
+
+        artistData = await Promise.all([
+            db.query(
+                `INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *`, 
+                [
+                    'Nevermind',
+                    'rock',
+                ]),
+        ]);
+            artist = artistData.map(({ rows }) => rows[0]);
+            singleArtistId = artist[0].id;
+
+        albumData = await Promise.all([
+            db.query(
+                'INSERT INTO Albums (name, year, artistId) VALUES ($1, $2, $3) RETURNING *',
+            [
+                'Nevermind',
+                1991,
+                singleArtistId
+            ]),
+            db.query(
+                'INSERT INTO Albums (name, year, artistId) VALUES ($1, $2, $3) RETURNING *',
+            [
+                'Bleach',
+                1989,
+                singleArtistId
+            ]),
+            db.query('INSERT INTO Albums (name, year, artistId) VALUES ($1, $2, $3) RETURNING *', 
+            [
+                'In Utero',
+                1993,
+                singleArtistId
+            ]),
+        ])
+
+        album = albumData.map(({ rows }) => rows[0]);
+    })
+
+    describe('PATCH /albums/{id}', () => {
+        it('updates the album and returns the updated record', async () => {
+            const { status, body } = await request(app)
+              .patch(`/albums/${album[0].id}`)
+              .send({ name: 'new album', year: 1994, artistId: singleArtistId
```

Comment:
As you are not changing the artistId here there is no point in sending it in the body for the patch request. Additionally you haven't written any code to handle a change to the artist Id. It would be good if you write that code and the alter this test to check it works